internal/lsp/completion.go

```diff
@@ -48,12 +48,14 @@ func toCompletionItem(candidate lang.Candidate, caps lsp.CompletionClientCapabil
 		kind = lsp.EnumCompletion
 	case lang.MapCandidateKind, lang.ObjectCandidateKind:
 		kind = lsp.StructCompletion
+	case lang.TraversalCandidateKind:
+		kind = lsp.VariableCompletion
 	}
 
 	// TODO: Omit item which uses kind unsupported by the client
 
 	var cmd *lsp.Command
-	if candidate.TriggerSuggest {
+	if candidate.TriggerSuggest && snippetSupport {
```

Comment:
This is probably something that should've been done some time ago. We do not include the `=` in the simplified TextEdits and so triggering suggest in that case is kind of pointless.

It is not a big deal for most clients today which probably do support snippets anyway, but it also helps reducing the amount of test data where in test we opt into snippet support only where we need it.